prompts/prompts.service.js

```diff
@@ -100,6 +100,14 @@ class PromptsService {
     };
   }
 
+  get pathPart() {
+    return {
+      type: 'input',
+      name: 'pathPart',
+      message: 'Path part for API Gateway (e.g. /chatbot)',
```

Comment:
I'd use API Gateway terminology instead of path part here. I think if you renamed this resource or something similar it would be much more descriptive.